tests/components/fritzbox/test_light.py

```diff
@@ -59,6 +59,23 @@ async def test_setup(hass: HomeAssistant, fritz: Mock) -> None:
     assert state.attributes[ATTR_MAX_COLOR_TEMP_KELVIN] == 6500
 
 
+async def test_setup_non_color(hass: HomeAssistant, fritz: Mock) -> None:
```

Comment:
please also test the different `supported_color_modes` based on different properties of the light entiy